semantics_analysis/relation_extractor/ontology_utils.py

```diff
@@ -983,9 +983,9 @@
                 'description': 'модель не используется в данном приложении',
                 'examples': [
                     {
-                        'text': 'Это было сделано путем fine-tune (точной настройки) предварительно обученного трансформера HuggingFace на собранных данных Genius.',
-                        'Model': 'трансформера HuggingFace',
-                        'Application': 'Genius'
+                        'text': 'Примером данного решения является использование парафрайзера на основе “rut5-base-paraphraser” из библиотеки huggingface.',
+                        'Model': '“rut5-base-paraphraser”',
+                        'Application': 'huggingface'

```

Comment:
This model is used in the application for a distribution, so other developers could build their own products using this model.

https://huggingface.co/cointegrated/rut5-base-paraphraser

The semantic meaning of `is used` can be different from my expectations, but still we may need to put some more precise example here.